### PR TITLE
make key_manager standalone

### PR DESCRIPTION
**Pull request by micolous**
First start on #309 

- [x] cargo test has been run and passes
- [x] documentation has been updated with relevant examples (if relevant)

### CODE REVIEW

File: fido-key-manager/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[package]
+name = "fido-key-manager"
+version = "0.1.0"
+authors = ["Michael Farrell <micolous+git@gmail.com>"]
+categories = ["authentication"]
+description = "Management CLI for FIDO/CTAP 2.x authenticators"
+edition = "2021"
+keywords = ["ctap", "fido", "webauthn"]
```
Comment (Firstyear) [inline fido-key-manager/Cargo.toml:8]: We should probably start sticking passkeys in the keywords just to get that sweet sweet SEO on crates.io.

File: fido-key-manager/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+* NFC support requires [PC/SC Lite][], and a PC/SC initiator (driver) for your
+  NFC transceiver (reader).
+
+  If you're using a transceiver with an NXP PN53x-series chipset (eg: ACS
+  ACR122, Sony PaSoRi), you will need to block the `pn533` and `pn533_usb`
+  kernel modules (which are incompatible [all other NFC software][linuxnfc])
+  from loading:
+
+  ```sh
+  echo "blacklist pn533" | sudo tee -a /etc/modprobe.d/blacklist.conf
```
Comment (Firstyear) [inline fido-key-manager/README.md:82]: Might be good to name the modprobe.d file differently here for making it clear that it's nfc or webauthn related.

File: fido-key-manager/README.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  echo "blacklist pn533_usb" | sudo tee -a /etc/modprobe.d/blacklist.conf
+  sudo rmmod pn533
```
Comment (Firstyear) [inline fido-key-manager/README.md:84]: modprobe -r, not rmmod. rmmod is the rm -f of kernel modules and not an good idea.

File: fido-key-manager/README.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  NFC transceiver (reader).
+
+  If you're using a transceiver with an NXP PN53x-series chipset (eg: ACS
+  ACR122, Sony PaSoRi), you will need to block the `pn533` and `pn533_usb`
+  kernel modules (which are incompatible [all other NFC software][linuxnfc])
+  from loading:
+
+  ```sh
+  echo "blacklist pn533" | sudo tee -a /etc/modprobe.d/blacklist.conf
+  echo "blacklist pn533_usb" | sudo tee -a /etc/modprobe.d/blacklist.conf
```
Comment (Firstyear) [inline fido-key-manager/README.md:83]: On some distros you *may* need to rebuild your initrd for this to take effect at next boot. On OpenSUSE/Fedora/SUSE/RHEL/Centos this is "sudo dracut -f".

**Review by Firstyear — CHANGES_REQUESTED** [review]

**Comment by yaleman** [comment]
Those CI failures look like you'll have to add `libpcsclite-dev` to `build_and_test`

File: fido-key-manager/Cargo.toml (hunk 1 above)
Comment (micolous) [reply fido-key-manager/Cargo.toml]: Done

File: fido-key-manager/README.md (hunk 2 above)
Comment (micolous) [reply fido-key-manager/README.md]: Done

File: fido-key-manager/README.md (hunk 3 above)
Comment (micolous) [reply fido-key-manager/README.md]: Done

File: fido-key-manager/README.md (hunk 4 above)
Comment (micolous) [reply fido-key-manager/README.md]: I didn't have this problem on Debian, and I don't know what circumstances that kernel module could be included in the initrd. I've added a note to that effect if it is an issue.

**Review by Firstyear — APPROVED** [review]

**Pull request merged**